Makefile

```diff
@@ -1,6 +1,6 @@
 .PHONY: docker
 build-docker:
-	docker build -t conpot docker/
+	docker build -t conpot:latest .
```

Comment:
A quick `grep` reveals that `docs/source/installation/quick_install.rst` also mentions the old `docker/` directory.